scripts/zones/Lufaise_Meadows/mobs/Padfoot.lua

```diff
@@ -26,12 +26,14 @@ entity.onMobDespawn = function(mob)
     local mobId = mob:getID()
 
     if mobId == ID.mob.PADFOOT[GetServerVariable("realPadfoot")] then
+        local respawnTime = (75600 + math.random(0, 6) * 1800) -- 21 to 24 hours with half hour windows
+
         for _, v in pairs(ID.mob.PADFOOT) do
             if v ~= mobId and GetMobByID(v):isSpawned() then
                 DespawnMob(v)
             end
 
-            xi.mob.nmTODPersist(GetMobByID(v), 75600 + math.random(0, 6) * 1800) -- 21 to 24 hours with half hour windows
```

Comment:
Was this original line not working?

Reply:
When camping Padfoot on live, it appears there is a discrepancy between TODs for the real and fake Padfoots, resulting in offset windows. If you test the original code locally, you can also notice this. For example: I killed each fake Padfoot, and then the real Padfoot last: 

![image](https://user-images.githubusercontent.com/112238268/229963934-100cab8f-daa5-41c2-9400-3ee743096f98.png)

And this example: I killed the real Padfoot first: 
![image](https://user-images.githubusercontent.com/112238268/229965134-9b051fa6-95f8-4d1b-8796-1eb14e97989e.png)

Both the real and fake Padfoots should pop at the same time.

Reply:
Oh good catch we had the random each separate one in the for loop